go/vt/mysqlctl/backupstats/defaults.go

```diff
@@ -0,0 +1,52 @@
+package backupstats
+
+import (
+	"vitess.io/vitess/go/stats"
+)
+
+var (
+	workLabels = []string{"Component", "Implementation", "Work"}
+
+	backupDuration       *stats.Gauge
+	backupWorkDurations  *stats.CountersWithMultiLabels
+	restoreDuration      *stats.Gauge
+	restoreWorkDurations *stats.CountersWithMultiLabels
+)
+
+func init() {
+	backupDuration = stats.NewGauge(
```

Comment:
These stats get reported no matter what, which doesn't quite line up with the way the rest of this PR is meant to works. Rethinnk this.